swarm.go

```diff
@@ -68,15 +68,15 @@ var (
 	uptimeGauge        = metrics.NewRegisteredGauge("stack.uptime", nil)
 )
 
-// the swarm stack
+// Swarm is an object abstracting the complete Swarm stack
```

Comment:
I think this comment is very unnecessary

Reply:
why? we're not increasing the number of lines, and the previous comment expressed very little.

Reply:
replaced with a middle-ground comment.

Reply:
The linter objects that exported objects should have a comment starting with `Swarm`...

Reply:
@holisticode good catch, changed to: `Swarm abstracts the complete Swarm stack` (don't think it's necessary to say `object` or `struct`)